docs/en/sql-reference/aggregate-functions/reference/varpop.md

```diff
@@ -4,79 +4,29 @@ slug: "/en/sql-reference/aggregate-functions/reference/varpop"
 sidebar_position: 32
 ---
 
-This page covers the `varPop` and `varPopStable` functions available in ClickHouse.
-
 ## varPop
 
-Calculates the population covariance between two data columns. The population covariance measures the degree to which two variables vary together. Calculates the amount `Σ((x - x̅)^2) / n`, where `n` is the sample size and `x̅`is the average value of `x`.
-
-**Syntax**
-
-```sql
-covarPop(x, y)
-```
-
-**Parameters**
-
-- `x`: The first data column. [Numeric](../../../native-protocol/columns.md)
-- `y`: The second data column. [Numeric](../../../native-protocol/columns.md)
-
-**Returned value**
-
-Returns an integer of type `Float64`.
-
-**Implementation details**
-
-This function uses a numerically unstable algorithm. If you need numerical stability in calculations, use the slower but more stable [`varPopStable`](#varpopstable) function.
-
-**Example**
+Calculates the population variance.
 
-Query:
-
-```sql
-DROP TABLE IF EXISTS test_data;
-CREATE TABLE test_data
-(
-    x Int32,
-    y Int32
-)
-ENGINE = Memory;
-
-INSERT INTO test_data VALUES (1, 2), (2, 3), (3, 5), (4, 6), (5, 8);
-
-SELECT
-    covarPop(x, y) AS covar_pop
-FROM test_data;
-```
-
-Result:
-
-```response
-3
-```
-
-## varPopStable
-
-Calculates population covariance between two data columns using a stable, numerically accurate method to calculate the variance. This function is designed to provide reliable results even with large datasets or values that might cause numerical instability in other implementations.
+:::note
+This function uses a numerically unstable algorithm. If you need [numerical stability](https://en.wikipedia.org/wiki/Numerical_stability) in calculations, use the [`varPopStable`](../reference/varpopstable.md) function. It works slower but provides a lower computational error.
```

Comment:
This is confusing.

The -Stable functions use a complex algorithm, similar to Kahan summation. They are rarely needed. For example, we don't recommend `sumKahan` instead of `sum`.

Reply:
Instead, let's write about numeric stability in the docs of the `-Stable` function and have no warning for ordinary functions.